fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3102,6 +3125,9 @@ public void dropMaterializedView(DropMaterializedViewStmt stmt) throws DdlExcept
             if (refreshTask != null) {
                 taskManager.dropTasks(Lists.newArrayList(refreshTask.getId()), false);
             }
+            if (view.isCloudNativeMaterializedView()) {
+                GlobalStateMgr.getCurrentState().getStorageVolumeMgr().unbindTableToStorageVolume(view.getId());
+            }
```

Comment:
`Database.dropTable` already has unbind operation.
better update the check in `Database.dropTable`.